build/vsts-ci.yml

```diff
@@ -174,13 +174,13 @@ phases:
   variables:
     BuildConfig: Release
     OfficialBuildId: $(BUILD.BUILDNUMBER)
-    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_CLI_TELEMETRY_OUTPUT: 1
```

Comment:
@safern told me this was a typo that needed to be fixed. I believe this change doesn't affects directly the issue that is being fixed in this PR.

Reply:
Changing this was a mistake. If you search for `DOTNET_CLI_TELEMETRY_OUTPUT` in all of github, you will find this is the only place this string is used. No one reads an `OUTPUT` environment variable.

But if you search for `DOTNET_CLI_TELEMETRY_OPTOUT` you will see it is used in hundreds of places.

Note: `OPTOUT` stands for "opt out" - as in "don't log CLI telemetry".

Reply:
You’re completely right I really got confused by reading output and just seems wrong and I recommended to change, my bad @antoniovs1029